packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
Right, it's not a question of whether you can use a Repository without a Model, but more about where we explain the concepts.  I think it's better to explain Repository concept here and link out to explanation of others elsewhere... However, for now it would be OK to leave the other material here in a "Concepts" section.  We can always reorganize the info later, which will certainly happen when we move the docs into the doc repo.